src/App.tsx

```diff
@@ -15,58 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [choosenProduct, setChoosenProduct] = useState<string>(`Jam`);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const onClickDeselect = () => {
```

Comment:
We call this function handlers so most often names of those function starts with `handleClick` .etc